deploy/examples/inventory-gcs-kubespray.example

```diff
@@ -0,0 +1,33 @@
+kube1 ansible_host=10.10.10.1
+kube2 ansible_host=10.10.10.2 gcs_disks='["/dev/sdb", "/dev/sdc", "/dev/sdd"]'
+kube3 ansible_host=10.10.10.3 gcs_disks='["/dev/sdb", "/dev/sdc", "/dev/sdd"]'
+kube4 ansible_host=10.10.10.3 gcs_disks='["/dev/sdb", "/dev/sdc", "/dev/sdd", "/dev/sde"]'
```

Comment:
`kube3` and `kube4` ansible_hosts are same here `10.10.10.3`

Reply:
@kshlm this is not fixed?